doc/arkode/guide/source/Usage/MRIStep_c_interface/Custom_Inner_Stepper/Description.rst

```diff
@@ -393,11 +398,9 @@ member functions:
    **Example codes:**
       * ``examples/arkode/CXX_parallel/ark_diffusion_reaction_p.cpp``
 
-Optional Member Functions
-"""""""""""""""""""""""""
+   .. versionchanged:: vX.X.X
```

Comment:
Can we specify this?
```suggestion
   .. versionchanged:: v6.0.0
```

Reply:
I've update `scripts/ updateVersion.sh` to find and update `x.x.x` or `X.X.X` in the documentation and change log in case we need to release with a different version number than currently planned. So they'll get set to the correct version number when creating the release branch.